mirascope/chat/models/openai_chat.py

```diff
@@ -96,7 +95,8 @@ def stream(
             A `OpenAIChatCompletionChunk` for each chunk of the response.
 
         Raises:
-            Re-raises any exceptions thrown by the openai chat completions create call.
+            Re-raises any exceptions thrown by the openai chat completions
```

Comment:
these sections I'm realizing are missing the `ValueError` thrown if neither `prompt` nor `messages` is not provided.

I'm thinking it might be worth creating our own MirascopeOpenAIError (like we discussed before) that wraps any error bubbled up? That way we can say something like:

```python
"""
Raises:
    ValueError: ...
    MirascopeOpenAIError: raises any OpenAI errors
"""
```

Reply:
dont think we need a MirascopeOpenAIError, when we can achieve the same thing with calling it OpenAIError and linking to their docs

Reply:
I added this line to all the openai class docstirngs
```
            OpenAIError: raises any OpenAI errors, see:
                https://platform.openai.com/docs/guides/error-codes/api-errors
```